mapsforge-map-android/src/main/java/org/mapsforge/map/android/rendertheme/AssetsRenderTheme.java

```diff
@@ -43,7 +43,7 @@ public AssetsRenderTheme(Context context, String relativePathPrefix, String file
 	public AssetsRenderTheme(Context context, String relativePathPrefix, String fileName, XmlRenderThemeMenuCallback menuCallback) throws IOException {
 		this.assetName = fileName;
 		this.relativePathPrefix = relativePathPrefix;
-		this.inputStream = context.getAssets().open(this.assetName);
+		this.inputStream = context.getAssets().open((this.relativePathPrefix == null) ? "" : this.relativePathPrefix + this.assetName);
```

Comment:
Here we parse the render theme xml file, i.e. ```fileName``` parameter which became ```assetName``` variable.

The file path can be totally described by this, so we don't need any mix with the ```relativePathPrefix``` which is for the resources paths in the xml itself.